5/5.6/Dockerfile

```diff
@@ -72,3 +73,11 @@ ENTRYPOINT ["docker-entrypoint.sh"]
 
 EXPOSE 3306
 CMD ["mysqld"]
+
+# Download source code for GPL packages.
+RUN echo "deb-src http://deb.debian.org/debian/ jessie main" >> /etc/apt/sources.list \
+	&& echo "deb-src http://repo.mysql.com/apt/debian/ jessie mysql-${MYSQL_MAJOR}" >> /etc/apt/sources.list \
+	&& apt-get update
+COPY download_source /usr/local/src/
+RUN /usr/local/src/download_source
+RUN rm /etc/apt/sources.list && rm /usr/local/src/download_source
```

Comment:
The script now doesn't modify this file anymore. It adds a new `.list` file containing `deb-src` entries copied from `deb` entries from `/etc/apt/sources.list` and all `.list` files under `/etc/apt/sources.list.d/` instead. Much cleaner.